docs/visualize_smpl.md

```diff
@@ -29,7 +29,7 @@
     If you want to visualize a T-pose smpl or your poses do not have global_orient, you can do:
     ```python
     import torch
-    from mmhuman3d.core.visualization import visualize_T_pose
+    from mmhuman3d.core.visualization. import visualize_T_pose
```

Comment:
Here should be `from mmhuman3d.core.visualization.visualize_smpl import visualize_T_pose` ?

Reply:
Fixed in [2dd48b7](https://github.com/open-mmlab/mmhuman3d/pull/190/commits/2dd48b700eca74ef1e54061ffc11d50bd581037d).